core/domain/base_model_validators.py

```diff
@@ -757,6 +757,8 @@ def _validate_commit_cmds_schema(cls, item):
                 continue
             try:
                 change_domain_object(commit_cmd_dict)
+            except utils.DeprecatedError:
```

Comment:
I think you can make this clearer -- perhaps utils.DeprecatedCommandError.

I agree with @vojtechjelinek that you should at least leave a comment here to explain what's going on.

Reply:
Done.